File: Javascript/twoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const arrayMay = new Map();
    for (let i = 0; i< nums.length; i++) {
        arrayMay.set(nums[i], i);
    }      
    for (let i = 0; i < nums.length; i++) {
        let tmp = target - nums[i];
        if (arrayMay.get(tmp) && arrayMay.get(tmp) != i) {
            return [i, arrayMay.get(tmp)]
        }
    }
};

File: Python/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        converted_set = {}
        for i in range(len(nums)):
            converted_set[nums[i]] = i
        for i in range(len(nums)):
            rest = target - nums[i]
            index = converted_set.get(rest)
            if index and index != i:
                return [index, i]